resources/views/partials/movie_list.blade.php

```diff
@@ -0,0 +1,18 @@
+<div class="row">
+    @forelse($movies as $item)
+        <div class="col-xs-12 col-md-3">
+            <div class="card" style="width: 100%; min-height: 40rem">
```

Comment:
Inlined styles? Why they should or should not be here?

Reply:
In these particular point having a class will be a overkill since i use only in here. But agree that in a real life project it can be better on a class, no doubt.